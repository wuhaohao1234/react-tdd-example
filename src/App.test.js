import { render } from '@testing-library/react';
import App from './App';

test('should generate a snapshot for app component', () => {
  const { asFragment } = render(<App />);
  expect(asFragment(<App/>)).toMatchSnapshot()
});
