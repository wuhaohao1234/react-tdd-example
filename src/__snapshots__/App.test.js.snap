// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate a snapshot for app component 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    hello world
  </div>
</DocumentFragment>
`;
